dbaas/logical/views.py

```diff
@@ -1760,6 +1760,11 @@ def database_backup(request, context, database):
         database.infra.backup_hour
     )
 
+    context['cant_clone_db'] = False
```

Comment:
This code can change for:
Remove the if and put directly
context['cant_clone_db'] = database.databaseinfra.disk_offering.size_gb() > clone_size
Dont forget to put a default if the configuration does not exist.
clone_size = Configuration.get_by_name_as_int('clone_size_limit', DEFAULT_HERE)